### DISCUSSION TITLE
Naming/Top1-accuracy inconsistency between LCE GitHub documentation and source files

### DISCUSSION THREAD

**Discussion by simonmaurer** [larq/compute-engine#662] [Q&A]
Hi

With respect to reporting new latency numbers for the QuickNet models on the Apple M1 platform I noticed the following:
In the source file of the [QuickNet](https://github.com/larq/zoo/blob/main/larq_zoo/sota/quicknet.py) models the reported Top1-Accuracies are different than the one stated in the [documentation](https://github.com/larq/compute-engine/blob/main/README.md).

Not to be picky, just to be consistent maybe we can update the naming to be consistent between documentation & source classes.

Is it correct that the following models coincide:
QuickNet (58.6%) <-> QuickNetSmall (59.4%)
QuickNetLarge (62.7%) <-> QuickNet (63.3%)
QuickNetXL (67%) <-> QuickNetLarge (66.9%)
?

**Comment by AdamHillier** [comment]
Hi @simonmaurer, thank you for pointing that out. You are absolutely right - we recently updated to new versions of the models, which are indeed named differently (and have slightly different accuracies). The version on the right is the 'correct' current version - we need to update our docs!

**Reply by simonmaurer** [reply]
perfect! I could update the ReadMe.md (as it is part of the GitHub repo) and add it in the next PR (macos_arm64 support) including numbers for the Apple mac mini (Apple M1, 1 and 4 threads)

**Reply by AdamHillier** [reply]
For now I've opened https://github.com/larq/compute-engine/pull/663 - we need to collect new benchmark numbers before we can merge it. But it would be great to add in latencies for the M1 mac once you've added support for it!

**Reply by simonmaurer** [reply]
alright. added the benchmark results in the [PR](https://github.com/larq/compute-engine/pull/664) and as a comment in #663
